resources/views/base.blade.php

```diff
@@ -36,12 +36,14 @@ function gtag(){dataLayer.push(arguments);}
 
         <script>
             // Listen for events dispatched from Livewire components...
-            Livewire.on('rules-filtered', () => {
-                document.querySelectorAll('pre code.language-diff').forEach((block) => {
-                    hljs.highlightBlock(block);
+            document.addEventListener('DOMContentLoaded', function () {
+                document.addEventListener('rules-filtered', () => {
+                    setTimeout(() => {
```

Comment:
I think https://developer.mozilla.org/en-US/docs/Web/API/Window/requestAnimationFrame is the recommended function to do something after rerender (see for some very interesting insights into the JS event loop https://www.youtube.com/watch?v=cCOL7MC4Pl0)

Reply:
@SanderMuller Thanks for sharing :+1: 
Indeed the `requestAnimationFrame()` does the refresh without blinking from pure text to hljs colored one :+1:

Reply:
@SanderMuller I've checked it and will use it as it solves the blinking, thanks for sharing :+1:

Reply:
@SanderMuller thanks for sharing it. I leaned something new and it's time to change some codebase of mine. ✅